packages/data/addon/utils/metric.ts

```diff
@@ -1,19 +1,27 @@
 /**
- * Copyright 2018, Yahoo Holdings Inc.
+ * Copyright 2020, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
 import { isEmpty } from '@ember/utils';
 import { get } from '@ember/object';
 
+type Parameters = Dict<string>;
+interface ColumnAttributes {
+  name: string;
+  parameters: Parameters;
+}
+interface MetricObject {
```

Comment:
We'll have to come back later to extract this out to something like this commit from Jon's earlier work

https://github.com/yahoo/navi/blob/a7b5bf862cd79dfd71c1f7cf7bea9564c7a9b26f/packages/data/types/global.d.ts#L17-L65

Reply:
Yeah we definitely need some global types for the most common shapes